requirements/dev.ini

```diff
@@ -3,6 +3,7 @@
 
 ansible==2.7.5
 django-debug-toolbar==1.11
+fabric3==1.14.post1
```

Comment:
Fabric3 1.14.post1 has been released Feb 11, 2018
Fabric 2.0.0 has been released May 10, 2018 with Python3 support.
I think it's better to use fabric==2.4.0 in this requiriments